src/attestation/Attestation.ts

```diff
@@ -21,6 +21,20 @@ import IPublicIdentity from '../types/PublicIdentity'
 
 const log = factory.getLogger('Attestation')
 
+export function compressAttestation(attestation: IAttestation): any[] {
```

Comment:
you could get rid of this any (and 3 others) by using `Array<IAttestation[keyof IAttestation]>`.

Reply:
Nice, I didn't know what to put, but I have never used this before. I have an idea what it is doing, but maybe explain this to me would be nice of you.

Reply:
`keyof IAttestation` is a type of all the keys of IAttestation. `IAttestation[keyof IAttestation]` represents the types of all values of IAttestation. I think. [Here are the docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#keyof-and-lookup-types).

Reply:
It's creating a union type of all the possible keys of IAttestation so this would be IAttestation["claimHash" | "cTypeHash" | "owner" | "delegationId" | "revoked"]